emcc.py

```diff
@@ -2076,7 +2076,10 @@ def phase_linker_setup(options, state, newargs):
     assert not settings.SIDE_MODULE
     if settings.MAIN_MODULE == 1:
       settings.INCLUDE_FULL_LIBRARY = 1
+    # Called from preamble.js once the main module is instantiated.
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$loadDylibs']
+    if settings.STACK_OVERFLOW_CHECK == 2:
+      settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$setDylinkStackLimits']
```

Comment:
Is this stack limit change related?

Reply:
This looks like part of the PR it's rebased on, but I see it isn't. Is it an old version of that PR's emcc changes maybe? Or why is it needed here?